README.rst

```diff
@@ -76,8 +76,49 @@ Run tests
 To check your code, run `make checks`. It will run style checks and tests. It's also possible to
 run sub-target independently:
 
-- `make checks-style` to run only style-related checks (linting).
-- `make tests` to run tests. Python 3.7 is used by default. But you can run tests with another version by
-  running: `PYTHON_TEST_VERSION=python3.x make tests`. You can also set this variable in the Makefile.
+- ``make checks-style`` to run only style-related checks (linting).
+- ``make tests`` to run tests. Python 3.7 is used by default. But you can run tests with another version by
+  running: ``PYTHON_TEST_VERSION=python3.x make tests``. You can also set this variable in the Makefile.
 
-After running tests, the coverage is available in the folder `coverage_report`.
+After running tests, the coverage is available in the folder ``coverage_report``.
+
+Use Oereblex data model
+---------
```

Comment:
We need the identical number of underline characters as the chapter title has, otherwise package building will fail